.github/workflows/tests.yaml

```diff
@@ -81,6 +81,36 @@ jobs:
           sudo npm install -g stylelint-config-standard
```

Comment:
should we remove the global install of the npm dependencies on linux as well?

Reply:
So far on Linux I have seen that the only NPM package that does not work with a global install is `eslint`. Until we do more testing to show that those tools work on a developer machine without a global install I would be in favor of leaving them as global installs.